.github/workflows/prod_deploy.yml

```diff
@@ -0,0 +1,90 @@
+name: Deploy to prod
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+jobs:
+  run_tests:
+    uses: ./.github/workflows/test.yml
+    secrets: inherit
+
+  build:
+    needs:
+      - run_tests
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Generate Image Tag
+        run: echo "IMAGE_TAG=364159549467.dkr.ecr.$AWS_REGION.amazonaws.com/stela:$([[ ${GITHUB_REF##*/} = main ]] && echo main || echo feature)-$(git rev-parse --short $GITHUB_SHA)" >> $GITHUB_ENV
```

Comment:
So we'll be able to publish a feature branch to prod? (That seems plausible in case of a hotfix but I'm mainly asking in case this was a copy paste error!)

Reply:
I think this is because we might want to send feature branches to *staging*, now that I've read the whole PR, but let me know if that's right.

Reply:
This shouldn't in practice run on non-`main` branches (and there are repo-level constraints in place to block the prod deploy if it is), but if it does we want to be sure that a build from a feature branch wouldn't get tagged with "main"